app/models/school.rb

```diff
@@ -80,7 +79,7 @@ class School < ApplicationRecord
 
   belongs_to :calendar, optional: true
   belongs_to :template_calendar, optional: true, class_name: 'Calendar'
```

Comment:
Are we planning to remove this now? Or later?

Reply:
Later... There's already a lot in this PR!